setup.py

```diff
@@ -44,6 +44,7 @@ def read(fname):
         'vw': ['vowpalwabbit==8.8.1'],
         'nn': ['tensorflow-cpu==2.3.1', 'lmdb==1.0.0'],
         'omikuji': ['omikuji==0.3.*'],
+        'yake': ['yake @ git+https://github.com/LIAAD/yake@v0.4.3'],
```

Comment:
Yake has just (again) been released on PyPI so this could be changed to `yake==0.4.3` or similar. See https://github.com/LIAAD/yake/issues/42